src/QAT/qat/purr/backends/verification.py

```diff
@@ -49,7 +54,7 @@ def __init__(self, qpu_version, verification_engine_type: type):
         self.version = qpu_version
 
 
-def verify_instructions(builder: InstructionBuilder, qpu_type: QPUVersion):
+def verify_instructions(builder: InstructionBuilder, qpu_type: QPUVersion, max_circuit_duration: int = 90000):
```

Comment:
Remove argument, max circuit duration is linked to hardware version.

Reply:
Done